organization-understanding.md

```diff
@@ -0,0 +1,76 @@
+# Memorandum of Understanding
+
+**This is a draft.**
+
+[NumFOCUS](http://numfocus.org/)
+will apply to Google Summer of Code (GSoC)
+as a umbrella organization.
+[All projects/organizations supported by NumFOCUS](http://numfocus.org/projects/)
+can participate at GSoC under NumFOCUS umbrella.
+Projects/organizations not supported by NumFOCUS
+can participate at GSoC under NumFOCUS umbrella
+if approved by NumFOCUS board.
+
+## NumFOCUS Administrator for GSoC
+
+The NumFOCUS board will indicate one person,
+named in this document as **organization administrator**,
+to be in charge of NumFOCUS application **for each edition** of GSoC.
+
+The **organization administrator** is responsible for
+
+-   create the initial application for NumFOCUS to apply for GSoC
+-   advertise the application to **all** projects/organization supported
+    by NumFOCUS
+-   manage NumFOCUS profile at GSoC
+-   request a minimum and a maximum number of slots
+    that can accomodate the slots requested by each project/organization
+    under NumFOCUS umbrella.
+
+## Coordinators for GSoC
+
+Each project/organization under NumFOCUS umbrella for GSoC
+can indicate up to three names,
+named in this document as **coordinators**,
+to help the **organization administrator** managing NumFOCUS application.
+
+**Coordinators** will have administrative access to
+NumFOCUS profile at GSoC.
+
+The **coordinators** are responsible for
+
+-   provide a list of projects ideas to **organization administrator**
+-   advertise the application to the project/organization they represent
+-   connect users with NumFOCUS profile
+-   request to **organization administrator** a minimum and a maximum number of
+    slots
+
+## Scoring Students Proposals
+
+Students proposals will be scoring following the description bellow.
+
+-   **0 stars**: must be reject.
+-   **1 star**: slot should be request and should be reject if not enough slots are received.
+-   **2 stars**: slot should be request
+-   **3 stars**: slot should be request and should be accepted even if few slots are received.
+
+**Coordinators** and **mentors** should vote **only** in proposals
+related with the project/organization they are involved.
+
+## Guidelines to Select Accept Students Proposals
+
+**Organization administrator** and **coordinators** will
+discuss and accept/reject students proposal based
+on the number of slots received and the score of each proposal.
+
+**If** NumFOCUS received few slots than projects/organizations
+under its umbrella,  **organization administrator** and
+**coordinators** will not accept more than one proposal
+per project/organization. In this case, **organization administrator**
+and **coordinators** will try to accept proposals
+related to projects that can only apply under NumFOCUS umbrella.
```

Comment:
I know we talked about this and it sounded ok when I first heard it, but it is an odd rule because there are lots of GSoC orgs and it is very likely that every sub-org could submit under other orgs. For example, Portland State University often has a wide range of different projects, and nothing ties them together beyond being open source.

I think we (coordinators) should simply discuss together how to divy out the slots once we see the applications and get the slots. They should go to the best applicants first and secondly we need to consider if the application has a mentor. For example, with SymPy we've gotten excellent applications but no mentors that want to tackle that project, so we accept the student and ask them if they will work on something different with a different mentor. Pairing students and mentors together appropriately, regardless of project, has higher chance of success.